manifests/server/config.pp

```diff
@@ -142,8 +142,11 @@
 
     ensure_packages([$package_name])
 
+    $exec_command = ['/usr/sbin/semanage', 'port', '-a', '-t', 'postgresql_port_t', '-p', 'tcp', shell_escape($port)]
+    $exec_unless = ["/usr/sbin/semanage port -l | grep -qw ${shell_escape($port)}"]
     exec { "/usr/sbin/semanage port -a -t postgresql_port_t -p tcp ${port}":
-      unless  => "/usr/sbin/semanage port -l | grep -qw ${port}",
+      command => $exec_command,
+      unless  => $exec_unless,
```

Comment:
I'm wondering if this is needed. Isn't port a `Stdlib::Port`? If so, the value is already bound to an integer in a specific range by the type system.

Reply:
thats a good point